tests/test_arg_parse.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+
+from nose.tools import eq_
+
+from hashin import parser
+
+
+def test_everything():
+    args = parser.parse_known_args(
+        ['hashin', 'reqs.txt', 'sha512', '--verbose', '--python-version', '3.5']
```

Comment:
This ought to fail or else it's trying to hash in packages called "reqs.txt" and "sha512".